experimental/sql/src/ssa_to_impl.py

```diff
@@ -105,8 +105,17 @@ class YieldRewriter(RelSSARewriter):
   @op_type_rewrite_pattern
   def match_and_rewrite(self, op: RelSSA.Yield, rewriter: PatternRewriter):
     # TODO: This is a hack to circumvent the FrozenList. Could this be done cleaner?
-    rewriter.replace_matched_op(
-        [RelImpl.Yield.get([o.op for o in op.operands])])
+    if isinstance(op.parent_op(), RelImpl.Select):
```

Comment:
So in essence: the problem has moved to over here.

Reply:
I feel that we can do the same thing for rel_ssa as well, but then we are taking about tuples on the rel_ssa level already which I feel is not great. Certainly, it also not the end of the world, but...

I think at some point, we will want to merge rel_ssa und rel_impl anyway, so this code won't be in the final product from how I see it. The final product will go from tree to reasoning about tuples directly, I feel.